frontend/src/modules/userManagement/views/addUserView.js

```diff
@@ -24,6 +28,9 @@ define(function(require){
       this.$('.field-error').each(function(index, element) {
         var $error = $(element);
         var $input = $error.siblings('input');
+        
+        // Password is no longer a sibling so needs special treatment
```

Comment:
Please could you remove this comment, somebody looking at this file after this PR isn't going to know that password ever used to be a sibling